frontend/package.json

```
@@ -1,5 +1,6 @@
 {
   "name": "tracertm-frontend",
+  "packageManager": "bun@1.1.38",
```

Biome (2.4.14)

[error] The key packageManager was already declared.

(lint/suspicious/noDuplicateObjectKeys)